560.js

```diff
@@ -0,0 +1,35 @@
+
+// Time Complexity : O(n^3)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : Time Limit Exceeded
```

Comment:
Try to optimize this. Try to extend the approach for the previous problem. Try to REMEMBER the number of times a sub array with a particular sum has occurred.